config/environment-variables.js

```diff
@@ -37,7 +37,7 @@ var config = {
   "views": {
         "locals":{
             piwikID: session.piwikId,
-            feedbackURL:live_variables.Public ? live_variables.feedbackURL : "https://www.gov.uk/done/get-document-legalised",
+            feedbackURL:live_variables.Public ? live_variables.feedbackURL : "https://www.smartsurvey.co.uk/s/legalisation1/",
```

Comment:
I would just totally remove the conditional statement and have it like:

feedbackURL:live_variables. feedbackURL

You can do a proper test locally and on Integration/Preprod that way